WebApplication/Migrations/20201123222800_DbBasexx.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace WebApplication.Migrations
+{
+    public partial class DbBasexx : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Buildings",
+                columns: table => new
+                {
+                    Id = table.Column<string>(nullable: false),
+                    SerialNumber = table.Column<string>(nullable: true),
+                    Name = table.Column<string>(nullable: true),
+                    Floors = table.Column<string>(nullable: true),
+                    Color = table.Column<string>(nullable: true),
+                    Shape = table.Column<string>(nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Buildings", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Equipments",
+                columns: table => new
+                {
+                    Id = table.Column<string>(nullable: false),
+                    SerialNumber = table.Column<string>(nullable: true),
+                    Name = table.Column<string>(nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Equipments", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Feedbacks",
+                columns: table => new
+                {
+                    SerialNumber = table.Column<string>(nullable: false),
+                    PatientId = table.Column<string>(nullable: true),
+                    Text = table.Column<string>(nullable: true),
+                    Date = table.Column<DateTime>(nullable: false),
+                    Approved = table.Column<bool>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Feedbacks", x => x.SerialNumber);
+                });
```

Comment:
What is the difference between  SerialNumber and Id in general?

Reply:
We haven't used serial nbrs, only ids. This is autogenerated file from migrations which should be deleted.